rfcs/0021-glossary-terms.md

```diff
@@ -0,0 +1,123 @@
+## Summary
+[summary]: #summary
+
+This RFC proposes standardized glossary terms for various concepts in the Coda Protocol and how they are communicated to the external world in documentation, logging, user interfaces, and public communications. This is not intended to cover the existing codebase in scope, as it may be too cumbersome at this point to rename everything to fit these proposed conventions. 
+
+However, it is advisable that over time the codebase also migrates to this shared terminology so that future community members can on-board seamlessly from high level documentation to contributing code.
+
+## Motivation
+
+[motivation]: #motivation
+
+Currently, multiple terms exist for several concepts in the Coda protocol, and they are used interchangeably when communicating to the world. This has some downsides as there is a lack of clarity on which terms to use, and could potentially cause confusion for people unfamiliar with the protocol. 
+
+As such, the motivation of this RFC is to standardize the naming convention for various concepts into a unified glossary that can be used in documentation, user-facing copy, and marketing channels. The expected outcome is unified language when communicating to the public and in developing Coda related products. This can then lead to more clarity and easier onboarding.
+
+## Detailed design
+
+[detailed-design]: #detailed-design
+
+The proposal contains the suggested term with it's associated concept it encapsulates, as well as alternative terms and the rationale for  the suggestion.
+
+The terms proposed are below:
+
+### Block Producer
+**Concept:** A consensus node in the network that is selected for a slot, and generates a block.
+
+**Rationale:** This term is used by other protocols, clearly describes the role duties, and is unambiguous.
+
+**Downsides:** EOS is the most popular chain that uses "block producers" and there is a chance of confusion if people think Coda also has only 21 block producers - but this is a tail risk.
+
+**Usage:** A block producer or block producing node stakes coda to participate in consensus.
```

Comment:
Suggest: "A block producer or block producing node actively attempts to produce new blocks on the blockchain by utilizing stake from accounts in the ledger."

This helps maintain clarity as to the fact that a block producing node is a specific kind of staker -- you can still be a staker and be staking on the network while not running your own personal block producing node.

Reply:
Will update this to reflect the fact that block producers can be delegated stake